Makefile

```diff
@@ -16,7 +16,8 @@ all: kops
 
 .PHONY: channels examples
 
-DOCKER_REGISTRY?=gcr.io/must-override
+#DOCKER_REGISTRY?=gcr.io/must-override
+DOCKER_REGISTRY=prashima
```

Comment:
Can this be taken as a env variable? Does not look good to checkin like this. Same comment applies for uncommenting the ssh command at line 197